openlibrary/macros/databarEdit.html

```diff
@@ -5,7 +5,7 @@
     <div id="editHistory">
         <a class="linkButton larger" href="$page.key?m=history" title="$_('View the editing history of this page')">$_("History")</a>
     </div>
-    <div class="editButton">
```

Comment:
This `editButton` class is on the "Cancel" button.
It has almost no effect, (other than a light aligning) and complicates detecting when to fetch the edit button.
So I just removed it.